src/tennisSimulator/TennisCommunity.java

```diff
@@ -0,0 +1,6 @@
+package tennisSimulator;
+
+public enum TennisCommunity {
```

Comment:
I don't like it. You are mixing two separate concepts, gender and organization. It is not bad to have two separate enums